src/org/labkey/test/components/ui/grids/FieldSelectionDialog.java

```diff
@@ -386,20 +386,25 @@ public FieldSelectionDialog removeFieldFromSelected(String field, int index)
     public FieldSelectionDialog removeAllSelectedFields()
     {
         List<WebElement> allItems = elementCache().getListItemElements(elementCache().selectedFieldsPanel);
+        boolean removedAll = true;
 
-        for(WebElement listItem : allItems)
+        for (WebElement listItem : allItems)
```

Comment:
Occasionally this method will fail because not every item was removed-- one theory I have is that it might be due to how they slide up when the previous one is removed.  It might be more reliable to iterate from the bottom up, if it's possible for the click to 'miss' if it happens when the target is moving